lib/rules/property-no-deprecated/index.mjs

```diff
@@ -103,6 +103,8 @@ const rule = (primary, secondaryOptions) => {
 
 			if (typeof mappedProp === 'undefined') return;
 
+			if (normalizedProp === '-webkit-box-orient' && value === 'vertical') return;
```

Comment:
Can you `toLowerCase()` the value before making the comparison?

Reply:
Done